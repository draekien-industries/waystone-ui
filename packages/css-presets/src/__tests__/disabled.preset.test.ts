import { disabledCss, disabledWithBackgroundCss } from '../disabled.presets';

describe('Disabled presets', () => {
  it('disabledCss should match snapshot', () => {
    expect(disabledCss).toMatchSnapshot();
  });

  it('disabledWithBackgroundCss should match snapshot', () => {
    expect(disabledWithBackgroundCss).toMatchSnapshot();
  });
});
