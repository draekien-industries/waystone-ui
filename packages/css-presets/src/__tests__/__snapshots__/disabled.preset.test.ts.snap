// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disabled presets disabledCss should match snapshot 1`] = `
{
  "color": "text-disabled",
  "cursor": "not-allowed",
}
`;

exports[`Disabled presets disabledWithBackgroundCss should match snapshot 1`] = `
{
  "backgroundColor": "muted",
  "color": "text-disabled",
  "cursor": "not-allowed",
}
`;
